Framework/MDAlgorithms/src/IntegrateEllipsoids.cpp

```diff
@@ -278,6 +278,16 @@ void IntegrateEllipsoids::init() {
       "IntegrateIfOnEdge", true,
       "Set to false to not integrate if peak radius is off edge of detector."
       "Background will be scaled if background radius is off edge.");
+
+  declareProperty("AdaptiveQBackground", false,
+                  "Default is false.   If true, "
+                  "BackgroundOuterRadius + AdaptiveQMultiplier * **|Q|**"
+                  "BackgroundInnerRadius + AdaptiveQMultiplier * **|Q|**");
```

Comment:
Have you checked how the double asterisks around the `|Q|` appear in the html documentation? At least [here](http://docs.mantidproject.org/nightly/algorithms/IntegratePeaksMD-v2.html) they seem a bit confusing to me.